src/top.rs

```diff
@@ -21,7 +21,7 @@ pub(crate) struct Topology {
     /// We don't use a Rt for async. Right now, we just use it to tick time
     /// forward in the same way we do it elsewhere. We'd like to represent
     /// network state with async in the future.
-    rt: Rt,
+    rt: Rt<'static>,
```

Comment:
This use of Rt was surprise to me and forced me to add Kind::NoSoftware but Kind::Client will work but I've thought adding explicit Kind is cleaner. `<'static>` hack is also awkward I had no idea it will work out. 

Using Rt for time only look strange to me in current state but it seems like we plan to extend it in future. Let me know if we need to change it.

Reply:
`'static` seems appropriate to me here.